TESTING_WITH_IAF-TEST.md

```diff
@@ -37,16 +38,37 @@ The module's test scenarios can be run manually with the Larva testtool. This wi
 4. In the Project Explorer, right-click the iaf-test module and select Properties. Go to Deployment Assembly, press [ Add... ]. Select Folder, press [ Next ]. Select the src/main/configurations folder, and press Finish. In the text field right of your new src/main/configurations item, enter `WEB-INF/classes/configurations`. _(might not be necessary anymore)_
 5. Do the same for the src/test/testtool folder. For that, enter `testtool` as deploy path. _(might not be necessary anymore)_
 
-## 3. Select database
+## 3. Create a Run Configuration - IntelliJ
+
+Create a Tomcat Run Configuration and set the following:
+1. Select your installed Tomcat Application Server
+2. Add the following parameters to the VM Options:
+   - `-Ddtap.stage=LOC`
+   - `-DauthAliases.expansion.allowed=testalias`
+   - `-Dlog.dir=$HOME/ibis4test-logs` (or whatever works for your system, drive letters on Windows must be lowercase).
```

Comment:
'whatever works for your system' is ook: wat werkt voor iaf-test.
Als je iaf-test larva testen maakt en/of aanpast, en daarbij afwijkende resultaten permanent accepteert door 'save result' te doen, en deze resultaten de log directory naam bevatten, dan kan dat veroorzaken dat het nieuw opgeslagen resultaat op andere machines weer faalt.
Om dat op te vangen is er een mechanisme met 'log.dir.match' en 'testdata.dir.match', maar dat werkt niet voor 100%: het gaat uit van een aantal bekende log.dir locaties. 

Als je log.dir op iets willekeurigs zet, zoals hier, dan kan je lokaal prima testen, maar als je resultaat files met die locatie incheckt, dan zullen die testen op andere machines falen.

Reply:
Ik zal dit aanpassen. Daarbij wil ik wel opmerken dat "c:/temp" 1) Heel erg Windows-specifiek is; "C:" en andere drive-letters bestaan niet op MacOS of Linux en 2) is een algemeen gebruikte temp-directory voor van alles en nog wat, volgens mij, dus kunnen er tussen de logs diverse andere dingen staan?

Als besproken in the standup, voor verbeteringen aan hoe Larva omgaat met test-data moet een nieuw ticket aangemaakt worden.